SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -229,6 +230,27 @@ void USpatialGameInstance::Init()
 	}
 }
 
+#if ENGINE_MINOR_VERSION >= 25
+AGameModeBase* USpatialGameInstance::CreateGameModeForURL(FURL InURL, UWorld* InWorld)
+#else
+AGameModeBase* USpatialGameInstance::CreateGameModeForURL(FURL InURL)
+#endif
+{
+#if ENGINE_MINOR_VERSION >= 25
+	AGameModeBase* GameMode = Super::CreateGameModeForURL(InURL, InWorld);
+#else
+	AGameModeBase* GameMode = Super::CreateGameModeForURL(InURL);
+#endif
```

Comment:
Nit: group these into one `#if` block with 3 lines of code in if and else blocks.